src/handlers/package.js

```diff
@@ -30,6 +33,31 @@ export const updatePackagesHandler = async (req, res) => {
 }
 
 
+export const notifypaymentHandler = async (req, res) => {
+    try {
+        const id = req.user.userId;
+        if (req.user.userType === "trainee") {
+            const insert = await insertPackageController(id, req.body)
+            if (insert) {
+                res.status(200).json({ successMessage: "The package was successfully inserted" })
+                return true;
+            }
+        }
+        if (req.user.userType === "trainer") {
```

Comment:
Move this to a seperate endpoint called listPaymentNotifications, with URL /packages/payments GET